bindings/python/htcondor2/_schedd.py

```diff
@@ -331,11 +334,19 @@ def jobEpochHistory(self,
         else:
             raise TypeError("since must be an int, string, or ExprTree")
 
+        ad_type = kwargs.get("ad_type", None)
+        if isinstance(ad_type, list):
+            ad_type = ",".join([str(i) for i in ad_type])
+        elif ad_type is None:
+            ad_type = ""
+        elif not isinstance(ad_type, str):
+            raise TypeError("ad_type must be a list of strings or a string")
```

Comment:
If we're going to say this, do we need `str(i)` on line 339?

(The v2 bindings don't currently check in a bunch of places if every element of a `List[str]` is a str, so you'd be in good company if you didn't.)